Content.Server/Explosion/TriggerSystem.cs

```diff
@@ -116,5 +143,23 @@ public void HandleTimerTrigger(TimeSpan delay, IEntity triggered, IEntity? user
                 Trigger(triggered, user);
             });
         }
+        public void SetProximityFixture(EntityUid uid, TriggerOnProximityComponent component, bool remove)
+        {
+            var entity = EntityManager.GetEntity(uid);
+            var broadphase = Get<SharedBroadphaseSystem>();
+
+            if (entity.TryGetComponent(out PhysicsComponent? physics))
+            {
+                var fixture = physics.GetFixture(component.ProximityFixture);
+                if (!remove && fixture != null)
+                {
+                    broadphase.DestroyFixture(physics, fixture);
+                }
+                else
+                {
+                    broadphase.CreateFixture(physics, new Fixture(physics, component.Shape) { CollisionLayer = (int) (CollisionGroup.MobImpassable | CollisionGroup.SmallImpassable | CollisionGroup.SmallImpassable), Hard = false, ID = component.ProximityFixture });
+                }
+            }
+        }
```

Comment:
Make SharedBroadphaseSystem a dependency.
```suggestion
            var broadphase = Get<SharedBroadphaseSystem>();

            if (ComponentManager.TryGetComponent<PhysicsComponent>(uid, out var physics))
            {
                var fixture = physics.GetFixture(component.ProximityFixture);
                if (!remove && fixture != null)
                {
                    broadphase.DestroyFixture(physics, fixture);
                }
                else
                {
                    broadphase.CreateFixture(physics, new Fixture(physics, component.Shape) { CollisionLayer = (int) (CollisionGroup.MobImpassable | CollisionGroup.VaultImpassable | CollisionGroup.Impassable | CollisionGroup.SmallImpassable), Hard = false, ID = component.ProximityFixture });
                }
            }
        }
```

Reply:
will this be called that much that it needs to be a dependency?

Reply:
Probably not I guess